rpm/0003-Move-large-array-out-of-stack.patch

```diff
@@ -0,0 +1,31 @@
+From 353ba68a1c1184aeea1355abe0c156f053362620 Mon Sep 17 00:00:00 2001
```

Comment:
This could be formatted as zero commit but fine like this as well.

Reply:
Fixed.